client/src/Components/App.jsx

```diff
@@ -1,20 +1,10 @@
 import React from 'react';
 import axios from 'axios';
-import styled from 'styled-components';
+import { StyledContainer } from '../Styles/style';
 import PhotosContainer from './PhotosContainer';
 import SaveButton from './SaveButton';
 import ShareButton from './ShareButton';
 
-const StyledContainer = styled.div`
-  display: grid;
-  grid-template-columns: 50% 25% 25%;
-  grid-template-rows: 50% 50%;
-  height: 475px;
```

Comment:
haha nice!! yea better to separate the styled components from the react components